scripts/deploy/all.sh

```diff
@@ -0,0 +1,13 @@
+# Deploy to all sites in parallel
+echo "🚢  Fetching from GitHub...";
+git fetch origin
+
+echo "🚢  Deploying...";
+yarn run concurrently \
```

Comment:
I didn't realize that `yarn run` could take shell scripts as arguments!

Reply:
This is super cool and useful, I'm surprised that yarn doesn't seem to document it very clearly: https://yarnpkg.com/lang/en/docs/cli/run/

Reply:
Yeah!  What's happening here is that `yarn run [script]` is running `concurrently` within the package.json env (like `bundle exec` in Ruby).  And that package, concurrently, happens to take a set of command line arguments that it should run concurrently.  This is how the `yarn start` command works too, to run the dev server and Rails concurrently.

Reply:
Ooooooooooooh 👍